src/tests/ftest/cart/threaded_server.c

```diff
@@ -108,6 +108,9 @@ int main(int argc, char **argv)
 	int                     rc;
 	int			i;
 	int                     thread_count;
+	crt_group_t		*grp;
+	char			*uri;
+
 
```

Comment:
if that's ok with you, could you please remove the changes to threaded_server? I'll make a separate PR with the changes I had made to that test.

Reply:
yup will do, this got included accidentally